src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -242,4 +242,40 @@ public void areUnsignedIntegersWithinRange() {
                 Integer.toString(i + range + 2), range));
     }
 
+    //---------------- Tests for areUnsignedIntegersWithinRange ---------------------------
```

Comment:
The comment is wrong hahah, should be `Tests for isIntegerLargerOrEqualToValue`

Reply:
oops typo .~.